config/autoload/dependencies.global.php

```diff
@@ -3,10 +3,8 @@
 return [
     'dependencies' => [
         'invokables' => [
-            App\Action\PingAction::class => App\Action\PingAction::class,
         ],
```

Comment:
I rather keep it __and__ put some nice comments to help users to learn about the framework, like in [middleware-pipeline.global.php](https://github.com/zendframework/zend-expressive-skeleton/blob/master/config/autoload/middleware-pipeline.global.php)